msteams-platform/tabs/how-to/create-tab-pages/configuration-page.md

```diff
@@ -27,6 +29,65 @@ An example of a configuration page is shown in the following image:
 
 The following code is an example of corresponding code for the configuration page:
 
+# [TeamsJS v2](#tab/teamsjs-v2)
+
+```html
+<head>
+    <script src='https://statics.teams.cdn.office.net/sdk/v1.6.0/js/MicrosoftTeams.min.js'></script>
+</head>
+<body>
+    <button onclick="(document.getElementById('icon').src = '/images/iconGray.png'); colorClickGray()">Select Gray</button>
+    <img id="icon" src="/images/teamsIcon.png" alt="icon" style="width:100px" />
+    <button onclick="(document.getElementById('icon').src = '/images/iconRed.png'); colorClickRed()">Select Red</button>
+
+    <script>
+        app.initialize();
+        let saveGray = () => {
+            microsoftTeams.settings.registerOnSaveHandler((saveEvent) => {
```

Comment:
for v2, I think this as well as lines 47, 57, and 58 should be pages.config instead of microsoftTeams.settings

Reply:
Should be corrected in latest commit: https://github.com/MicrosoftDocs/msteams-docs/pull/5884/commits/b4a65c5b699ccffc3bc647daaff38916f649c46d